Brauser.Common/Recipe/Recipe.cs

```diff
@@ -2,44 +2,83 @@ namespace Brauser.Common.Recipe;
 
 public record Recipe
 {
+    public Guid Id { get; set; }
     public string? Name { get; set; }
     public DateTimeOffset? Date { get; set; }
     public string? Type { get; set; }
 
-    public int? BatchSize { get; set; } // Ausschlagwürze
-    public int? BrewhouseYield { get; set; } // Sudhausausbeute
-    public double? OriginalGravity { get; set; } // Stammwürze
-    public int? Bitterness { get; set; } // Bittere
+    /// <summary>
+    /// Ausschlagwürze
+    /// </summary>
```

Comment:
I like it for now. We should remove this, after the converter is finished and we have completed the translation documents. For now, it's just for reference imho

Reply:
Yes, that's the plan.